rest-api/add-gif-picker/talkjs-giphy-integration/index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>TalkJS - Giphy Integration</title>
+    <meta name="description" content="">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet"
+        integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+    <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+    <script>
+        (function (t, a, l, k, j, s) {
+            s = a.createElement("script");
+            s.async = 1;
+            s.src = "https://cdn.talkjs.com/talk.js";
+            a.head.appendChild(s);
+            k = t.Promise;
+            t.Talk = {
+                v: 3,
+                ready: {
+                    then: function (f) {
+                        if (k)
+                            return new k(function (r, e) {
+                                l.push([f, r, e]);
+                            });
+                        l.push([f]);
+                    },
+                    catch: function () {
+                        return k && new k();
+                    },
+                    c: l,
+                },
+            };
+        })(window, document, []);
+    </script>
+    <script src="index.js" async defer></script>
+    <div class="chatbox-container" style="position: relative;">
+<div id="gif-search">
+    <form>
+        <div class="form-group" style="padding: 8px;">
+            <input id="searchTerm" type="text" class="form-control" autocomplete="off" placeholder="Search for GIFs">
+            <input type="submit" class="btn btn-outline-primary" style="margin-bottom: 5px;">
+            <div id="gifSearchResults"></div>
+        </div>
+    </form>
+</div>
+        <div id="talkjs-container" style="width: 90%; margin: 30px; height: 500px;">
+            <i>Loading chat...</i>
+        </div>
+    </div>
```

Comment:
You want the gif search and talkjs container to be inside the same div, and then you can display the gif-search on top. That will simplify styles.css so that you can just set the gif picker to 100% width instead of all the weird magic values you have currently.

Something like:

```html
<div style="width: 90%; margin: 30px; height: 500px; position: relative">
  <div id="gif-search" style="position: absolute; top: 0; left: 0; right: 0;"></div>
  <div id="talkjs-container" style="height: 100%; width: 100%"></div>
<div>
```

Where the inline styles can be moved into the css file

Reply:
I can do this for you if you want. I'm not sure I explained it properly

Reply:
Hey @stevenwaterman, the gif-search and talkjs-container are already inside the same div. 

```html
    <div class="chatbox-container">
        <div id="gif-search">
            <form>
                <input id="searchTerm" type="text" class="form-control" autocomplete="off"
                    placeholder="Search for GIFs">
                <input type="submit" class="btn btn-outline-primary" style="margin-bottom: 5px;">
                <div id="gifSearchResults"></div>
            </form>
        </div>
        <div id="talkjs-container">
            <i>Loading chat...</i>
        </div>
    </div>
 ```

Reply:
These are the styles I've used

```css
.chatbox-container{
    width: 90%; 
    margin: 30px; 
    height: 500px; 
    position: relative;
    border: 1px solid blue;
}
#talkjs-container{
    height: 100%; 
    width: 100%;
    border: 1px solid orange;
}
#gif-search{
    width: 100%;
    position: absolute; 
    top: 0; 
    left: 0; 
    right: 0;
    border: 1px solid red;
}
```

It still doesn't give the intended effect. Ignore the borders, I just used it to understand the widths separately. 

![image](https://github.com/talkjs/talkjs-examples/assets/42829077/80d331b2-eb33-4f3c-b734-880dff1fcdac)